srcCpp/CustomType.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __CUSTOMTYPE_H__
+#define __CUSTOMTYPE_H__
+/*
+ * (c) 2005-2018  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+#define Q(x) #x
+#define QUOTE(x) Q(x) // Used to set quotation marks
+
+#include QUOTE(RTI_CUSTOM_TYPE_FILE_NAME_SUPPORT)
+
+#include "perftest.h"
+#include "perftest_cpp.h"
+#include <string>
+
```

Comment:
Live cycle:
-call initialize sample
-if key call register for every key
for every sample that is sent, set_custom_type_data will be called
and in the end of the test finalize_custom_type_data will be called